### ISSUE TITLE
Can`t export mobilenetv3 model to onnx

### ISSUE THREAD

**Issue by jkparuchuri** [pytorch/vision#3463]
` model = models.mobilenet_v3_small(pretrained=True)
   input_np = np.random.uniform(0, 1, (1, 3, 224, 224))
   input_var = torch.FloatTensor(input_np)
   torch.onnx.export(model, args=(input_var), f="cnn.onnx", verbose=False, input_names=["input"], output_names=["output"])`

Error: RuntimeError: Exporting the operator hardsigmoid to ONNX opset version 9 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub.

**Comment by zhiqwang** [comment]
Hi

It seems that the `nn.Hardswish` caused this problem, actually the nightly version of PyTorch has addressed this problem, so the unit-test is pass.

If you are using PyTorch 1.7.x, you can replace it to an export friendly version of `Hardswish` as below and set ONNX opset version to 11.

```python
class Hardswish(nn.Module):
    """
    Export-friendly version of nn.Hardswish()
    """
    def __init__(self):
        super().__init__()

    def forward(self, x):
        return x * F.hardtanh(x + 3, 0., 6.) / 6.
```

**Comment by jkparuchuri** [comment]
@zhiqwang  I had tried on pytorch 1.8 and torchvision 0.9 released today; still I get the same error. Are you sure it got fixed already ?

`RuntimeError: Exporting the operator hardsigmoid to ONNX opset version 9 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub.`

**Comment by zhiqwang** [comment]
Hi @jkparuchuri 

You are right! I didn't test my proposal (Sorry, I only tested the method to solve the bug at `Hardswish` before). I've missed the `Hardsigmoid` in

https://github.com/pytorch/vision/blob/c991db82abba12e664eeac14c9b643d0f1f1a7df/torchvision/models/mobilenetv3.py#L35

As you mentioned, `torch.export` doesn't support this operator,

I replaced this operator with `F.hardtanh` to solve this problem (I've tested the whole model now)

You can do something as following to address this error

```diff
diff --git a/torchvision/models/mobilenetv3.py b/torchvision/models/mobilenetv3.py
index 1e2606d..fdf1f6d 100644
--- a/torchvision/models/mobilenetv3.py
+++ b/torchvision/models/mobilenetv3.py
@@ -32,7 +32,7 @@ class SqueezeExcitation(nn.Module):
         scale = self.fc1(scale)
         scale = self.relu(scale)
         scale = self.fc2(scale)
-        return F.hardsigmoid(scale, inplace=inplace)
+        return F.hardtanh(scale + 3, 0., 6., inplace=inplace) / 6.
 
     def forward(self, input: Tensor) -> Tensor:
         scale = self._scale(input, True)
```

Now there are two ways to solve the bug of this error,

1. Native support for the export of `Hardsigmoid`
2. Replace `F.hardsigoid` with `F.hardtanh` that is exported friendly and numerically equal as I do here

cc @datumbox @fmassa What is your suggestion?

**Comment by jkparuchuri** [comment]
Thank you @zhiqwang

**Comment by fmassa** [comment]
Another option could to register a custom ONNX op for `hardsigmoid` in https://github.com/pytorch/vision/blob/master/torchvision/ops/_register_onnx_ops.py , so that we can keep torchvision using the faster `hardsigmoid` while ONNX uses `hardtanh`.

@neginraoof what do you think it would be preferable? Are there plans on adding those missing operators to ONNX?

**Comment by ThomAub** [comment]
I agree with @fmassa and I think there is no need for a custom op for `HardSigmoid`.
I saw https://github.com/onnx/onnx/blob/master/docs/Operators.md#HardSigmoid so I might be mistaken but it's maybe just adding and registering this operator on the pytorch side ?

**Comment by zhiqwang** [comment]
Hi @fmassa , @ThomAub - It seems that https://github.com/pytorch/pytorch/issues/49649 and https://github.com/pytorch/pytorch/pull/54193 are working on this issue now.
